### PR TITLE
Prompt Cache Tokens in Prompt Deployment Executions Table

### PR DESCRIPTION
**Pull request by marlenc3**

### CODE REVIEW

File: fern/docs/content/change-log/2024-09.mdx [hunk 1]
```diff
@@ -2,6 +2,17 @@
 title: Changelog | September, 2024
 ---
 
+## New Prompt Caching Columns for Prompt Deployment Execution Table
+
+_September 23rd 2024_
+
+You can now see any Prompt Cache Read and Creation Tokens that are passed with your prompt deployment executions. This should allow
+you to analyse your prompt caching use, and fine tune to your expectations. Prompt Caching tokens are limited to Anthropic's
+Prompt Caching feature for now.
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-09.mdx]:
```suggestion
A while back Anthropic added support for [Prompt Caching](/changelog/2024/2024-08#prompt-caching-support-for-anthropic). With this update, you'll now see the number of Prompt Cache Read and Cache Creation Tokens used by a Prompt Deployment's executions if its backed by an Anthropic model.

This new monitoring data can be used to help analyze your cache hit rate with Anthropic and optimize your LLM spend.
```

**Review by noanflaherty — APPROVED** [review]

File: fern/docs/content/change-log/2024-09.mdx (hunk 1 above)
Comment (marlenc3) [reply fern/docs/content/change-log/2024-09.mdx]: this is so much better 🙇  changing then should be ready

Comment (marlenc3) [reply fern/docs/content/change-log/2024-09.mdx]: should be added!

File: fern/docs/content/change-log/2024-09.mdx [hunk 2]
```diff
@@ -2,6 +2,16 @@
 title: Changelog | September, 2024
 ---
 
+## New Prompt Caching Columns for Prompt Deployment Execution Table
+
+_September 23rd 2024_
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-09.mdx:7]: [nit] we keep forgetting commas lol

**Review by dvargas92495 — COMMENTED** [review]
✅

**Pull request merged**